README.md

```diff
@@ -125,6 +129,32 @@ The last "unscoped" strategy means there's no file- or
 component-local way of deciding if a style *could* be used elsewhere. You should
 not use sCiSSors on packages / projects with that strategy.
 
+## Using `scissors/cssmirroring_transformer`
+
+Required: Css janus.
+          Run the following command in shell:
+            mkdir -p $HOME/bin && svn checkout http://cssjanus.googlecode.com/svn/trunk/ $HOME/bin && export PATH=$HOME/bin/cssjanus:$PATH
+
+Example: see [example/mirroring](https://github.com/google/dart-scissors/tree/master/example/mirroring).
+
+`pubspec.yaml`:
+
+  ```
+  dev_dependencies:
+    scissors
+  transformers:
+  - scissors/cssmirroring_transformer
+  ```
+
+Valid settings:
+- `mirrorCss`: `true` by default in release mode and false by default in debug mode.
+- `cssDirection` `ltr` by default in debug and release mode.
+- `cssJanusPath`: `cssJanus` by default.
+
+###Limitations
+ - Works only on css files. Pre-processors like sass and less are not supported. 
+ - Does not handle directives like KeyFrames and page.
```

Comment:
links to spec please, and/or syntax snippets (e.g. `@keyframe`, '@page'), as `page` alone won't ring many bells